.github/workflows/pre-commit-check.yaml

```diff
@@ -4,19 +4,29 @@ on:
   push:
     branches:
       - master
+      - develop
   pull_request:
 
 jobs:
   build:
     runs-on: macOS-latest
     steps:
-    - uses: actions/checkout@v1
-
-    - name: Install prerequisites
-      run: |
-        brew install tfenv tflint terraform-docs pre-commit
-        pre-commit install
-        tfenv install
-    - name: pre-commit run all
-      run: |
-        pre-commit run -a
+      - uses: actions/checkout@v1
+      - name: Install prerequisites
+        run: ./bin/install-macos.sh
+      - name: initiallize Terraform
+        run: terraform init --backend=false
+      - name: debug
+        run: tflint -v
```

Comment:
do we want to leave debug in it? I'm fine with it, GH Actions are free for public repos

Reply:
I think it's fine to print the version there. Can't be bothered to remove it